mempool/src/mempool.rs

```diff
@@ -690,6 +693,21 @@ impl MempoolState {
 
         Some(tx)
     }
+
+    // Removes all the transactions that belong to some specific sender
```

Comment:
This does not remove transactions _to_ the account, which I would say also _belong_ to it.
```suggestion
    // Removes all the transactions sent by some specific address
```